docs/guides/spmd_api.md

```diff
@@ -0,0 +1,429 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+<!-- #region id="2-JYevEklHfB" -->
+# Scale up Flax modules with `pjit`
```

Comment:
Suggestion:

Maybe mention multi-host/multi-devices in the title so that users know what this scaling up with `pjit` means without indepth JAX knowledge. Similar to "Using JAX in multi-host and multi-process environments" https://jax.readthedocs.io/en/latest/multi_process.html